sys/linux/socket_netlink_route.txt

```diff
@@ -69,6 +74,9 @@ netlink_msg_route [
 	ipv6_newrule		netlink_msg[RTM_NEWRULE, fib_rule_hdr[AF_INET6], fib6_rule_policy]
 	ipv6_delrule		netlink_msg[RTM_DELRULE, fib_rule_hdr[AF_INET6], fib6_rule_policy]
 	ipv6_getrule		netlink_msg[RTM_GETRULE, fib_rule_hdr[AF_INET6], void]
+	ipv6_newaddrlabel	netlink_msg[RTM_NEWADDRLABEL, ifaddrlblmsg[AF_INET6], ifal_policy]
```

Comment:
I don't know if it makes sense to define `ifaddrlblmsg` as a template since all the uses have `AF_INET6` as the argument, but I'll leave this up to @dvyukov .